components/gitpod-protocol/src/protocol.ts

```diff
@@ -199,8 +202,6 @@ export interface AdditionalUserData {
     knownGitHubOrgs?: string[];
     // Git clone URL pointing to the user's dotfile repo
     dotfileRepo?: string;
-    // Identifies an explicit team or user ID to which all the user's workspace usage should be attributed to (e.g. for billing purposes)
-    usageAttributionId?: string;
```

Comment:
What will happen with existing records which do have this field set on the User?

Reply:
Briefly talked to @geropl about this. Given the small impact in terms of user already using this, a data migration seems to be not necessary. Please correct me, if this assumption is wrong, or if you see real problems on production.

Reply:
That's okay. As long as we won't fail to parse the JSON stored in `additionalData` of the user record then this seems okay.